test/eval/README.md

```
@@ -0,0 +1,100 @@
+| Variable | Default | Description |
+|----------|---------|-------------|
+| `EVAL_MODEL` | `claude-opus-4-6` | Model for agent invocation |
+| `EVAL_JUDGE_MODEL` | `claude-opus-4-6` | Model for judging |
+| `EVAL_RUNS` | `1` | Number of trials per scenario |
+| `EVAL_THRESHOLD` | `0.8` | Minimum pass rate (0.0-1.0) |
+| `EVAL_FOCUS` | | Ginkgo focus filter (substring match) |
+| `EVAL_VERBOSE` | | Set to `1` for verbose agent output |
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)